ddl/ddl_api.go

```diff
@@ -716,6 +727,25 @@ func getDefaultValue(ctx sessionctx.Context, col *table.Column, c *ast.ColumnOpt
 	return v.ToString()
 }
 
+func handleSequenceDefaultValue(c *ast.ColumnOption) (expr string, isExpr bool, err error) {
+	var sb strings.Builder
+	restoreFlags := format.RestoreStringSingleQuotes | format.RestoreKeyWordLowercase | format.RestoreNameBackQuotes |
+		format.RestoreSpacesAroundBinaryOperation
+	restoreCtx := format.NewRestoreCtx(restoreFlags, &sb)
+	switch x := c.Expr.(type) {
```

Comment:
Only one case? Then it would be better to use "if"

Reply:
addressed.